calculator.c

```diff
@@ -0,0 +1,43 @@
+/*Rumeet Goradia - Calculator Program*/
```

Comment:
Really great job, Rumeet, but this program wasn't in your final submission - I had to go look for it in your past commits. You must have accidentally deleted it in one of your commits. Be careful next time.